compendium/test_bundles/TBDynamicRefPolicy/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+usFunctionCreateDSTestBundle(TBDynamicRefPolicy)
```

Comment:
I didn't see TBDynamicRefPolicy sub-directory added to .. level CMakeList.txt.
Just wanted to make sure that TBDynamicRefPolicy wasn't added to CMakeLists.txt on purpose.

Reply:
good catch. this is an old test bundle that I forgot to delete.